tutorials/managing-gcp-projects-with-terraform/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Managing GCP Projects with Terraform
+description: Learn how to manage projects within an organization and project resources with Terraform.
+author: danisla
+tags: Terraform
+date_published: --
+---
+
+This tutorial demonstrates how to create and manage projects on Google Cloud Platform with Terraform. With Terraform, many of your resources such as projects, IAM policies, networks, Compute Engine instances, and Container Engine clusters can be managed, versioned, and easily recreated for your organization or teams. The state that Terraform generates is saved to Google Cloud Storage for persistence.
+
+Incremental changes made from the cloud console or gcloud command-line tool are difficult to track and recreate if needed later down the road. With Terraform, many of your resources like Projects, IAM policies, Networks, Compute Instances, and Container Engine Clusters can be managed and versioned and easily recreated for your organization or teams. The state that Terraform generates is saved to Google Cloud Storage for persistence.
```

Comment:
This paragraph largely duplicates the previous paragraph and should be deleted. (I think this is a new issue, but apologies if I missed this on the first pass.)